tests/test_adjoint_jacobian.py

```diff
@@ -1327,29 +1329,23 @@ def test_integration_custom_wires(returns):
 
     def circuit(params):
         circuit_ansatz(params, wires=custom_wires)
-        return qml.expval(returns), qml.expval(qml.PauliY(custom_wires[1]))
+        return np.array([qml.expval(returns), qml.expval(qml.PauliY(custom_wires[1]))])
```

Comment:
Does `qml.expval(returns), qml.expval(qml.PauliY(custom_wires[1])) `work?Do users have to change their python code to get this feature?

Reply:
I see `casted_to_array_def` was removed